bin/jobsub_cmd

```diff
@@ -85,6 +87,33 @@ def main() -> None:
         if i.startswith("--"):
             i = i[1:]
 
+        if i in [
+            "-format",
+            "-xml",
+            "--json",
+            "-autoformat",
+            "-io",
+            "-batch",
+            "-nobatch",
+            "-dag",
+            "-long",
+        ]:
+            default_formatting = False
+
+        if (
+            i
+            in [
+                "-constraint",
+                "-factory",
+                "-autocluster",
+                "-constraint",
+                "-unmatchable",
+                "-allusers",
+            ]
+            or i[0] in "0123456789"
```

Comment:
Is this for the formatting directives like `%0.1s` or whatever those are?

Reply:
no, that's for ClusterId's -- if they give us a 1234.0 or 1234.0@schedd that's a constraint for a single job, so we don't want to give the (jobsub_q default) group constraint.

Reply:
I see.  That makes sense.  Thanks for clearing that up for me.